<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wait</class>
 <widget class="QWidget" name="wait">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>345</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 255, 255);</string>
  </property>
  <widget class="QLineEdit" name="ID">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>请输入你的用户名...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="YES">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>确 认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EXIT">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>退 出</string>
   </property>
  </widget>
  <widget class="QLabel" name="A">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>105</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QLabel" name="B">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QLabel" name="C">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QLabel" name="D">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QLabel" name="E">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QLabel" name="F">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>GET READY</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GET READY</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GET READY</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GET READY</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="btn5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GET READY</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GET READY</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
